spec/services/analytics_spec.rb

```diff
@@ -18,7 +19,7 @@
     it 'identifies the user and sends the event to the backend' do
       user = build_stubbed(:user, uuid: '123')
 
-      analytics = Analytics.new(user, FakeRequest.new)
+      analytics = Analytics.new(user: user, request: FakeRequest.new, sp: 'http://localhost:3000')
```

Comment:
service_provider from L10 isn't in scope here, right? (learning)

Reply:
It is. This verifies that the SP that is passed in as an argument is the same SP that appears in `events.log` via the Ahoy gem.